assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -89,6 +89,7 @@ function FramesLayer() {
       tabIndex="-1"
     >
       <FramesPageArea
+        showOverflow
```

Comment:
Where should we put it? https://github.com/google/web-stories-wp/commit/d533bec7f6adc8c411391acf48c3604af113a762

Reply:
Either way, imho. Let me know what you'd prefer. But I generally like them in markup when the meaning is "logical state". And for layers, the overflow could be considering a logical state.

Reply:
After I saw [this](https://github.com/google/web-stories-wp/blob/32042bd166b66c5fce41ff2e1204d440faaf049b/assets/src/edit-story/components/canvas/layout.js#L156) my thought was: if the value is fixed, put it there.

Reply:
Sounds good. Done.